frontend/public/components/dashboard/dashboards-page/dashboards.tsx

```diff
@@ -6,69 +6,20 @@ import { Helmet } from 'react-helmet';
 import { useTranslation } from 'react-i18next';
 
 import { ClusterDashboard } from './cluster-dashboard/cluster-dashboard';
-import { HorizontalNav, PageHeading, LoadingBox, Page, AsyncComponent } from '../../utils';
-import Dashboard from '@console/shared/src/components/dashboard/Dashboard';
-import DashboardGrid, {
-  GridPosition,
-  GridDashboardCard,
-} from '@console/shared/src/components/dashboard/DashboardGrid';
-import {
-  useExtensions,
-  DashboardsCard,
-  DashboardsTab,
-  isDashboardsCard,
-  isDashboardsTab,
-} from '@console/plugin-sdk';
+import { HorizontalNav, PageHeading, LoadingBox } from '../../utils';
 import { RootState } from '../../../redux';
 
-const getCardsOnPosition = (cards: DashboardsCard[], position: GridPosition): GridDashboardCard[] =>
-  cards
-    .filter((c) => c.properties.position === position)
-    .map((c) => ({
-      Card: () => <AsyncComponent loader={c.properties.loader} />,
-      span: c.properties.span,
-    }));
-
-const getPluginTabPages = (tabs: DashboardsTab[], cards: DashboardsCard[]): Page[] =>
-  tabs.map((tab) => {
-    const tabCards = cards.filter((c) => c.properties.tab === tab.properties.id);
-    return {
-      href: tab.properties.id,
-      name: tab.properties.title,
-      component: () => (
-        <Dashboard>
-          <DashboardGrid
-            mainCards={getCardsOnPosition(tabCards, GridPosition.MAIN)}
-            leftCards={getCardsOnPosition(tabCards, GridPosition.LEFT)}
-            rightCards={getCardsOnPosition(tabCards, GridPosition.RIGHT)}
-          />
-        </Dashboard>
-      ),
-    };
-  });
```

Comment:
Export these and use it in OCS package.

Reply:
alright... i will keep this code here only and import these function in `ocs-dashboards.tsx` file that i created under `ceph-storage-plugin`...

Reply:
ack... I will export these functions from here and import them in ocs-dashboards.tsx file that I created under `ceph-storage-plugin` directory...